src/renderer/public/localization/locales/es/translation.json

```diff
@@ -1,15 +1,18 @@
 {
+  "alwaysOnTop": "Always on Top?",
   "attemptingToConnectToF1MV": "Intentando conectar con F1MV en:",
   "attemptingToConnectToLiveReplayTimingWindow": "Intentar conectarse a una ventana de tiempo en directo o de repetición:",
   "attemptingToGetCurrentSessionInfoFromF1mv": "Intento de obtener información de la sesión actual de F1MV...",
   "back": "Atrás",
   "checkNetworkSettings": "Mueva el ratón a la esquina inferior derecha y haga clic en el engranaje de configuración para comprobar la configuración de red.",
   "currentSession": "Sesión en Curso:",
   "digiflagNotLinked": "DigiFlag no está vinculado actualmente a F1MV",
+  "dutch": "Holandés",
   "english": "Inglés",
   "failedToConnect": "Fallo de conexión con F1MV",
   "failedToRetrieveCurrentSession": "Fallo al recuperar la sesión actual",
   "french": "Francés",
+  "general": "General",
```

Comment:
To be translated?

Reply:
No that’s actually correct I was surprised too